SPECS/librepo/librepo.spec

```diff
@@ -28,7 +28,7 @@ BuildRequires:	gpgme-devel
 BuildRequires:	openssl-devel
 BuildRequires:	attr
 
-Requires:       openssl-devel
+Requires:       openssl
```

Comment:
Please move unversioned .so files to devel package and it should remove the runtime depenency on openssl-devel package